py/foundation/in_memory.py

```diff
@@ -121,6 +122,9 @@ def id(self) -> str:
 
   def get_children(self) -> Iterable[Word]:
     yield from (c for c in self._children if isinstance(c, Word))
+  
+  def get_children_with_spaces(self) -> Iterable[Union[Word, Whitespace]]:
+    yield from (c for c in self._children if isinstance(c, Word) or isinstance(c, Whitespace))
```

Comment:
How are you imagining an external consumer using this? Would this make sense as an internal message, or maybe having `InMemorySpatialText._get_in_memory_text` directly access `InMemoryText._children` instead because it knows its `_lines` elements are `InMemoryText` objects?

Reply:
This is pretty much solely for internal use, particularly by the 2D splicing method. Should I just add a `_` to the front of it to indicate that?